internal/common/apps.go

```diff
@@ -5,30 +5,50 @@ import (
 	"time"
 
 	"github.com/giantswarm/apiextensions-application/api/v1alpha1"
-	"github.com/giantswarm/clustertest/pkg/wait"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime/pkg/client"
 
+	"github.com/giantswarm/clustertest/pkg/logger"
+	"github.com/giantswarm/clustertest/pkg/wait"
+
 	"github.com/giantswarm/cluster-test-suites/internal/state"
 )
 
 func runApps() {
 	Context("default apps", func() {
 		It("all default apps are deployed without issues", func() {
+			skipDefaultApps, err := state.GetCluster().UsesUnifiedClusterApp()
+			Expect(err).NotTo(HaveOccurred())
 
 			// We need to wait for default-apps to be deployed before we can check all apps.
 			defaultAppsAppName := fmt.Sprintf("%s-%s", state.GetCluster().Name, "default-apps")
 
-			Eventually(wait.IsAppDeployed(state.GetContext(), state.GetFramework().MC(), defaultAppsAppName, state.GetCluster().Organization.GetNamespace())).
-				WithTimeout(30 * time.Second).
-				WithPolling(50 * time.Millisecond).
-				Should(BeTrue())
+			if skipDefaultApps {
```

Comment:
`skipDefaultApps` sounds like default apps aren't deployed. Maybe `skipDefaultAppsApp` (not much less confusing, I know)?

[minor since this code is temporary]

Reply:
I think I'd rather this code checked if a `default-apps-XXX` existed on the MC and based on that decide the following code paths.

Reply:
> skipDefaultApps sounds like default apps aren't deployed. Maybe skipDefaultAppsApp (not much less confusing, I know)?

A typo I think (it was `skipDefaultAppsApp` in other places), fixed it now.